packages/tooltip/src/components/popover/Tippy.module.css

```
@@ -133,4 +133,39 @@
   .tippy-box[data-theme~="light"] > .tippy-svg-arrow {
     fill: #fff;
   }
+
+  /*Dark mode!*/
+  .tippy-box[data-theme~="dark"] {
+    color: #26323d;
+    box-shadow: 0 0 20px 4px rgba(154, 161, 177, 0.15),
+      0 4px 80px -8px rgba(36, 40, 47, 0.25),
+      0 4px 4px -2px rgba(91, 94, 105, 0.15);
+    background-color: var(--lhds-color-ui-900);
```

Specen säger att det ska vara ui-800 på alla ställen.

```suggestion
    background-color: var(--lhds-color-ui-800);
```